arviz/plots/traceplot.py

```diff
@@ -16,7 +16,9 @@ def plot_trace(
     figsize=None,
     textsize=None,
     lines=None,
+    compact=False,
```

Comment:
I agree this should be `False` by default, we should also have a max number of plots by default.